product-service/src/handlers/getProductsList.ts

```diff
@@ -1,45 +1,47 @@
 import { APIGatewayProxyHandler, APIGatewayProxyResult } from 'aws-lambda';
 import 'source-map-support/register';
-import productList from '../productList.json';
-import type { ProductList } from '../models/product';
-import getAgeByName from '../services/agify/methods/getAgeByName';
+import { Client } from 'pg';
+import dbOptions from '../dbConfig';
+
+console.log('DB CONFIG::', dbOptions);
+
+const getAllProductsQuery = `
+select 
+  p.id, p.title, p.description, p.price, s.count 
+from 
+  products p 
+left join stocks s 
+ ON id = product_id
+`;
+
+const defaultHeaders = {
```

Comment:
[proposal]
I see that code was refactored concerning the headers and I propose to go further and extract defaultHeaders constant to separate place -> reusability between all lambdas (now we repeat the same code)